src/main/java/seedu/duke/common/InfoMessages.java

```diff
@@ -20,6 +20,12 @@ public enum InfoMessages {
     INFO_LIST_UNFILTERED("There are no transaction records that match your search expression."),
     INFO_STATS_EMPTY("There are no statistics available yet for the given statistics type."),
     INFO_STATS_CATEGORIES("Here are the total savings for each category:"),
+    INFO_STATS_MONTHS("Here is the expenditure summary for each month:"),
+    INFO_STATS_MONTHS_COMMENT_ONE("Excellent! You saved quite a lot this month."),
+    INFO_STATS_MONTHS_COMMENT_TWO("Wow, keep up the good work. You saved at least 80% of your income."),
+    INFO_STATS_MONTHS_COMMENT_THREE("Good effort, you saved at least half of your income."),
+    INFO_STATS_MONTHS_COMMENT_FOUR("Hmm, it spent quite a sum. Try saving more in future."),
+    INFO_STATS_MONTHS_COMMENT_FIVE("You spent too much. Try to manage your expense within your budget."),
```

Comment:
It seems that these comments are displayed based on the proportion of their expense/income. Do consider more straightforward naming such as INFO_STATS_MONTHS_VERY_HIGH_SAVING, INFO_STATS_COMMENTS_HIGH_SAVING, INFO_STATS_COMMENTS_MEDIUM_SAVING instead of ONE, TWO, THREE.

Reply:
Yes, thank you for this feedback. They have been renamed as in this PR: https://github.com/AY2223S1-CS2113-W12-2/tp/pull/134